app/forms/hyrax/hyrax_form.rb

```diff
@@ -11,8 +13,6 @@ def notable_rights_statement?
     def self.multiple?(field)
```

Comment:
I think it's only necessary for `identifier`, which needs weird behavior because of expectations in Grocer (we should fix that, but I didn't wanna cross gem boundaries for this)